beta-src/src/components/ui/WDPopover.tsx

```diff
@@ -21,21 +22,18 @@ const WDPopover: React.FC<WDPopoverProps> = function ({
   children,
   isOpen,
   onClose,
+  open,
```

Comment:
if you are clicking to open, you should be passing that as a function bound to the popoverTrigger...

Reply:
will add it to the popoverTrigger